epub33/tts/index.html

```diff
@@ -159,19 +159,33 @@ <h3>The <code>ssml:ph</code> attribute</h3>
 					</dd>
 				</dl>
 
-				<p>This attribute inherits the requirements of the [[SSML]] <code>phoneme</code> element's <a
+				<p>The <code>ssml:ph</code> attribute inherits the authoring requirements of the [[SSML]]
+						<code>phoneme</code> element's <a
 						href="https://www.w3.org/TR/2010/REC-speech-synthesis11-20100907/#S3.1.10"><code>ph</code></a>
-					attribute, with the following additions:</p>
+					attribute.</p>
 
-				<ul class="conformance-list">
-					<li>
-						<p id="ssml-ph-concat">When the <code>ssml:ph</code> attribute appears on an element that has
-							text node descendants, the corresponding document text to which the pronunciation applies is
-							the string that results from concatenating the descendant text nodes, in document order. The
-							specified phonetic pronunciation MUST therefore logically match the element's textual data
-							in its entirety (i.e., not just an isolated part of its content).</p>
-					</li>
-				</ul>
+				<p id="ssml-ph-concat">When the <code>ssml:ph</code> attribute appears on an element that has text node
+					descendants, the corresponding document text to which the pronunciation applies is the string that
+					results from concatenating the descendant text nodes, in document order. The specified phonetic
+					pronunciation must therefore logically match the element's textual data in its entirety (i.e., not
+					just an isolated part of its content).</p>
+
+				<p id="ssml-ph-text">EPUB Creators SHOULD NOT use the <code>ssml:ph</code> attribute on elements without
+					text content that a Text-to-Speech engine would normally render (e.g., on empty <code>div</code> or
+						<code>span</code> elements). The attribute is not intended for adding additional voicing only
+					for users listening to TTS playback and Reading Systems are expected to ignore the attribute if it
+					does not replace text they would normally render.</p>
+
```

Comment:
I had some difficulties to parse the English sentence (as someone not an expert in TTS). Maybe a comma would help as in

> [...] TTS playback, and Reading Systems are expected [...]

But maybe also

> The attribute is only intended to add voicing for users listening to TTS playback, and [...]

But I may completely misunderstand the sentence which may be a sign of parsing difficulties...

Reply:
Ya, the first part of the sentence if a bit wordy. I've tried to shorten it and also added a comma to separate from the RS explanation.